packages/FormElement/stories/examples/Showcase.js

```diff
@@ -13,6 +13,7 @@ const getRootKnobs = () => ({
 
 const getLabelKnobs = () => ({
   labelText: text("label", "Field label"),
+  tooltipAlignment: select("Direction", ["top", "bottom", "left", "right"], "top"),
```

Comment:
instead of Direction should we just call it `tooltipAlignment `? 🤔

Reply:
Yeah, the label in the Storybook UI should ideally match the prop name, if possible.